javaHapiValidatorLambda/src/main/java/software/amazon/fwoa/Validator.java

```diff
@@ -64,22 +61,24 @@
     private final FhirContext ctx;
 
     private final String fhirVersion;
-    private final String implementationGuidesFolder;
+    private final List<IGObject> implementationGuidesIndices;
+    private final List<IGObject> implementationGuidesResources;
 
     public Validator() {
```

Comment:
If the `List<IGObject> DEFAULT_IMPLEMENTATION_GUIDES` is null now, would the first two constructors still be valid?